### DISCUSSION TITLE
Persist pagination page index in queryString/session

### DISCUSSION THREAD

**Discussion by m-pernkopf** [filamentphp/filament#10302] [General]
Dear Community,

in my filament 2 application I try to persist a few parameters in a table, f.ex Filter, Search, ColumnSort and Pagination page.
I do this by using the shouldPersistTablexxxx functions, which works great. The only thing I am not able to persist is the page index

f.ex: I do some search, sort and filter stuff in my table and switch to page 2, which gives me a queryString like
_?tableSortColumn=radgroup_id&tableSortDirection=asc&tableSearchQuery=der&tableFilters[group][client_id]=1005&page=2_

When I switch to another resource and return back the queryString looks like this
_?tableFilters[group][client_id]=1005&tableSortColumn=radgroup_id&tableSortDirection=asc&tableSearchQuery=der_ **(but not &page)**

How can I enable the page index also to be persisted in the session?

Regards, Michael

**Comment by jhonoryza** [comment]
@danharrin can we achieved this in filament v3 ? or should we create pull request ?

**Reply by danharrin** [reply]
You may create a pull request if you wish
